library/crates/Cargo.toml

```diff
@@ -9,6 +9,16 @@ path = "" # ignored by cargo raze
 antlr-rust = "0.2.0"
 rocksdb = "0.17.0"
 cxx = "1.0.59"
+futures = "0.3.21"
+
+# gRPC dependencies
```

Comment:
These dependencies, raze arguments and aliased targets are taken from the `rules_rust` Protobuf setup guide at https://bazelbuild.github.io/rules_rust/rust_proto.html.

The `Cargo.toml` file that `rules_rust` recommends using is located at https://github.com/bazelbuild/rules_rust/blob/f3a34f67e97a12aaf3adaa9cfa957dceb9ad7b89/proto/raze/Cargo.toml.

We've adapted it slightly for our needs, notably by pinning the `log` version to `0.4.6`.